core/store/src/migrations.rs

```diff
@@ -71,18 +70,22 @@ where
     store_update.finish()
 }
 
-pub fn migrate_28_to_29(store_opener: &StoreOpener) -> anyhow::Result<()> {
-    let store = store_opener.open().unwrap().get_store(crate::Temperature::Hot);
-    let mut store_update = store.store_update();
-    store_update.delete_all(DBCol::_NextBlockWithNewChunk);
-    store_update.delete_all(DBCol::_LastBlockWithNewChunk);
-    store_update.commit()?;
-
-    set_store_version(&store, 29)?;
+/// Migrates database from version 28 to 29.
+///
+/// Deletes all data from _NextBlockWithNewChunk and _LastBlockWithNewChunk
+/// columns.
+pub fn migrate_28_to_29(storage: &crate::NodeStorage) -> std::io::Result<()> {
```

Comment:
this perhaps should stay anyhow?

Reply:
Dunno.  I switched it to io when in previous version StoreOpenerError was parametrised on error and thus near-store could be made not to depend on anyhow.  And now I kinda like that the error conversion is pushed out of the store’s migration functions.